src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,30 @@
+import './TodoInfo.scss';
+
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = (
+  {
+    todo: {
+      title,
+      completed,
+      user,
+    },
+  },
+) => (
```

Comment:
this is good new way and old one was also right one!

Advantages of Destructuring:
It makes developer’s life easy, by assigning their own variables.
Nested data is more complex, it takes time to access, but by the use of destructuring, we can access faster of nested data.
It improves the sustainability, readability of code.
It helps to cut the amount of code used in an application.
It trims the number of steps taken to access data properties.

nothing wrong in this but
so i think this is still better way,

```suggestion
export const TodoInfo: React.FC<Props> = ({
    title,
    completed,
    user,
  }) => (
```